build/bench/Dockerfile

```diff
@@ -87,13 +91,12 @@ RUN bash -c "bench --version"
 # !!! UPDATE NODEJS PERIODICALLY WITH LATEST VERSIONS !!!
 # https://nodejs.org/en/about/releases/
 # https://nodejs.org/download/release/latest-v10.x/
-# https://nodejs.org/download/release/latest-v12.x/
-# https://nodejs.org/download/release/latest-v13.x/
-ENV NODE_VERSION=12.20.0
-ENV NODE_VERSION_FRAPPEV11=10.23.0
+# https://nodejs.org/download/release/latest-v14.x/
+ENV NODE_VERSION=14.17.0
+ENV NODE_VERSION_FRAPPEV11=10.24.1
```

Comment:
node.js v10 is EOL already, ref: https://github.com/nodejs/Release/#end-of-life-releases, maybe should also update it to v12 (consider to keep two versions of node.js here, ref: https://github.com/frappe/frappe_docker/issues/123#issuecomment-596211931)? I can send a pull request if it make sense to you.

Reply:
I think it's needed for old v11 development.
ERPNext V11 won't work without nodejs v10.

Reply:
Oh, okay, I got it. Thanks.